package.json

```
@@ -39,7 +39,8 @@
     "dev": "NODE_ENV=development BROWSERSYNC_OPEN=true gulp dev",
     "docs:mod": "node tasks/mod-extractor.js",
     "lint:components": "node ./tasks/packageLint.js",
-    "mv:preview": "rimraf dist/preview && mv -f tools/preview/storybook-static dist/preview",
+    "lint:styles": "stylelint ./components/*/dist/**/*.css",
```

It looks like the path here is partially incorrect. As it's written, it's _only_ checking against CSS files that are in a sub-directory in each component's `dist`, and if I try to force a linting error, nothing is being detected by Stylelint. 

I think what we want is:

```suggestion
    "lint:styles": "stylelint ./components/*/dist/*.css",
```

---

Woah! Nice catch. I always forget if ** is 0+ or 1+.